projects/packages/my-jetpack/src/class-rest-product-data.php

```diff
@@ -47,67 +47,6 @@ public static function permissions_callback() {
 		return current_user_can( 'manage_options' );
 	}
 
-	/**
-	 * Gets the product data for all products
-	 *
-	 * @return array|WP_Error
-	 */
-	public static function get_all_product_data() {
-		$site_id        = \Jetpack_Options::get_option( 'id' );
-		$wpcom_endpoint = sprintf( 'sites/%d/jetpack-product-data?locale=%2$s&force=wpcom', $site_id, get_user_locale() );
-		$api_version    = '2';
-		$response       = Client::wpcom_json_api_request_as_blog( $wpcom_endpoint, $api_version, array(), null, 'wpcom' );
-		$response_code  = wp_remote_retrieve_response_code( $response );
-		$body           = json_decode( wp_remote_retrieve_body( $response ) );
-
-		$capabilities = self::get_backup_capabilities();
-
-		if ( is_wp_error( $response ) || empty( $response['body'] ) || 200 !== $response_code ) {
-			return new WP_Error( 'site_products_data_fetch_failed', 'Site products data fetch failed', array( 'status' => $response_code ? $response_code : 400 ) );
-		}
-
-		// If site has backups and the realtime backup capability, add latest undo event
-		if (
-			in_array( 'backup-realtime', $capabilities->data['capabilities'], true )
-		) {
-			$body->backups->last_undoable_event = self::get_site_backup_undo_event();
-		}
-
-		return rest_ensure_response( $body, 200 );
-	}
-
-	/**
-	 * Get an array of backup/scan/anti-spam site capabilities
-	 *
-	 * @access public
-	 * @static
-	 *
-	 * @return WP_Error|\WP_REST_Response|null An array of capabilities
-	 */
-	public static function get_backup_capabilities() {
```

Comment:
This was originally used to see if the user's plan included realtime backups so that we didn't show anything funky if they had backup but not realtime. However after looking through this code, and remembering that we have a default state for if there are no undoable events, I don't think it is needed any longer. Nice cleanup